src/opentype.mjs

```diff
@@ -3,135 +3,46 @@
-import { isNode } from './util.js';
-import Font from './font.js';
-import Glyph from './glyph.js';
-import { CmapEncoding, GlyphNames, addGlyphNames } from './encoding.js';
-import parse from './parse.js';
-import BoundingBox from './bbox.js';
-import Path from './path.js';
-import cpal from './tables/cpal.js';
-import colr from './tables/colr.js';
-import cmap from './tables/cmap.js';
-import cff from './tables/cff.js';
-import stat from './tables/stat.js';
-import fvar from './tables/fvar.js';
-import gvar from './tables/gvar.js';
-import cvar from './tables/cvar.js';
-import avar from './tables/avar.js';
-import hvar from './tables/hvar.js';
-import glyf from './tables/glyf.js';
-import gdef from './tables/gdef.js';
-import gpos from './tables/gpos.js';
-import gsub from './tables/gsub.js';
-import head from './tables/head.js';
-import hhea from './tables/hhea.js';
-import hmtx from './tables/hmtx.js';
-import kern from './tables/kern.js';
-import ltag from './tables/ltag.js';
-import loca from './tables/loca.js';
-import maxp from './tables/maxp.js';
-import _name from './tables/name.js';
-import os2 from './tables/os2.js';
-import post from './tables/post.js';
-import meta from './tables/meta.js';
-import gasp from './tables/gasp.js';
-import svg from './tables/svg.js';
-import { PaletteManager } from './palettes.js';
+import { tinf_uncompress as inflate } from './tiny-inflate@1.0.3.esm.mjs'; // from code4fukui/tiny-inflate-es
+import Font from './font.mjs';
+import Glyph from './glyph.mjs';
+import { CmapEncoding, GlyphNames, addGlyphNames } from './encoding.mjs';
+import parse from './parse.mjs';
+import BoundingBox from './bbox.mjs';
+import Path from './path.mjs';
+import cpal from './tables/cpal.mjs';
+import colr from './tables/colr.mjs';
+import cmap from './tables/cmap.mjs';
+import cff from './tables/cff.mjs';
+import stat from './tables/stat.mjs';
+import fvar from './tables/fvar.mjs';
+import gvar from './tables/gvar.mjs';
+import cvar from './tables/cvar.mjs';
+import avar from './tables/avar.mjs';
+import hvar from './tables/hvar.mjs';
+import glyf from './tables/glyf.mjs';
+import gdef from './tables/gdef.mjs';
+import gpos from './tables/gpos.mjs';
+import gsub from './tables/gsub.mjs';
+import head from './tables/head.mjs';
+import hhea from './tables/hhea.mjs';
+import hmtx from './tables/hmtx.mjs';
+import kern from './tables/kern.mjs';
+import ltag from './tables/ltag.mjs';
+import loca from './tables/loca.mjs';
+import maxp from './tables/maxp.mjs';
+import _name from './tables/name.mjs';
+import os2 from './tables/os2.mjs';
+import post from './tables/post.mjs';
+import meta from './tables/meta.mjs';
+import gasp from './tables/gasp.mjs';
+import svg from './tables/svg.mjs';
+import { PaletteManager } from './palettes.mjs';
 /**
  * The opentype library.
  * @namespace opentype
  */
 
-// File loaders /////////////////////////////////////////////////////////
```

Comment:
I thought we agreed that this would still function in 2.0.0 with a warning, and be removed in a subsequent release.